packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -31,13 +30,11 @@ function arrowGenerator() {
       width: '2em',
       height: '1em',
       '&::before': {
-        flip: false,
```

Comment:
I suspect we don't need the flip properties inside the `'&::before': {` selectors. Could you test it?

Reply:
> @j-mendez The property is targeted at https://github.com/alitaheri/jss-rtl. Did you face a specific issue with it?

yes, this creates invalid css when using the component. If I try to validate my html I get errors from this.

Reply:
Ah, we need to fix that. We didn't catch it in our main page: https://validator.w3.org/nu/?doc=https%3A%2F%2Fmaterial-ui.com%2Fcomponents%2Ftooltips%2F.

Reply:
@oliviertassinari  in reference to the error on there, if you move `<Head>` to your _app then make sure your custom doc looks like this https://nextjs.org/docs/advanced-features/custom-document. Make sure to have a empty <Head /> on your `_document.js` file

Reply:
Ok, figure why you can reproduce the issue and material-UI doesn't. Material-UI adds the jss-rtl plugin, while you are using the default set of plugins that doesn't.

Reply:
@oliviertassinari should I include the plugin locally or will this be included into the next build?

Reply:
I don't know. I'm curious to hear @eps1lon point of view. For instance, we could add a custom plugin, with the bare minimum to drop this property.

Reply:
I have found an alternative, we can dodge the issue by disabling rtl on the whole stylesheet.